spec/traits.dd

```diff
@@ -2,7 +2,102 @@ Ddoc
 
 $(SPEC_S Traits,
 
-$(HEADERNAV_TOC)
+$(COMMENT Group traits logically rather than automatic lexical order)
+$(DIVC page-contents quickindex,
+$(DIVC page-contents-header,
+    $(B Contents)
+)
+$(OL
+$(LI $(RELATIVE_LINK2 grammar, Grammar))
+$(NAV_ITEMS Types,
+    $(NAV_SUB_ITEM isArithmetic)
+    $(NAV_SUB_ITEM isFloating)
+    $(NAV_SUB_ITEM isIntegral)
+    $(NAV_SUB_ITEM isScalar)
+    $(NAV_SUB_ITEM isUnsigned)
+    $(NAV_SUB_ITEM isAssociativeArray)
+    $(NAV_SUB_ITEM isStaticArray)
+    $(NAV_SUB_ITEM isCopyable)
+    $(NAV_SUB_ITEM isPOD)
+    )
+
+$(NAV_ITEMS Functions,
+    $(NAV_SUB_ITEM getFunctionAttributes)
+    $(NAV_SUB_ITEM getFunctionVariadicStyle)
+    $(NAV_SUB_ITEM isDisabled)
+    $(NAV_SUB_ITEM isStaticFunction)
+    $(NAV_SUB_ITEM isReturnOnStack)
+    )
+
+$(NAV_ITEMS Function Parameters,
+    $(NAV_SUB_ITEM getParameterStorageClasses)
+    $(NAV_SUB_ITEM isRef)
+    $(NAV_SUB_ITEM isOut)
+    $(NAV_SUB_ITEM isLazy)
+    )
+
+$(NAV_ITEMS Symbols,
+    $(NAV_SUB_ITEM getAttributes)
+    $(NAV_SUB_ITEM getLocation)
+    $(NAV_SUB_ITEM getVisibility)
+    $(NAV_SUB_ITEM getProtection)
+    $(NAV_SUB_ITEM identifier)
+    $(NAV_SUB_ITEM isDeprecated)
+    $(NAV_SUB_ITEM isFuture)
+    $(NAV_SUB_ITEM isNested)
+    $(NAV_SUB_ITEM isSame)
+    $(NAV_SUB_ITEM isTemplate)
+    )
+
+$(NAV_ITEMS Aggregates,
+    $(NAV_SUB_ITEM allMembers)
+    $(NAV_SUB_ITEM child)
+    $(NAV_SUB_ITEM hasCopyConstructor)
+    $(NAV_SUB_ITEM hasMember)
+    $(NAV_SUB_ITEM hasPostblit)
+    $(NAV_SUB_ITEM getAliasThis)
+    $(NAV_SUB_ITEM getMember)
+    $(NAV_SUB_ITEM getOverloads)
+    $(NAV_SUB_ITEM getUnitTests)
```

Comment:
I see the text considers a module an aggregate "Takes one argument, a symbol of an aggregate (e.g. struct/class/module).", I thought an aggregate was a class, struct or union, so I would put this under Symbols.

Reply:
OK, but for consistency I have also moved the following to Symbols as they apply to modules too:
* allMembers
* hasMember
* getOverloads
* getUnitTests
* parent